SayTheirNames/Source/Controller/Petitions/PetitionTableViewCell.swift

```diff
@@ -0,0 +1,218 @@
+            bannerImageView.centerYAnchor.constraint(equalTo: imageViewContainer.centerYAnchor),
+            bannerImageView.centerXAnchor.constraint(equalTo: imageViewContainer.centerXAnchor),
+            bannerImageView.widthAnchor.constraint(greaterThanOrEqualTo: imageViewContainer.widthAnchor),
+            
+            verifiedLabel.trailingAnchor.constraint(equalTo: imageViewContainer.trailingAnchor, constant: -Self.VerifiedTextMargin),
+            verifiedLabel.topAnchor.constraint(equalTo: imageViewContainer.topAnchor, constant: Self.VerifiedTextMargin),
+            
+            // the titleLabel should always be below the verified button AND the banner image
+            titleLabel.topAnchor.constraint(greaterThanOrEqualToSystemSpacingBelow: verifiedLabel.bottomAnchor, multiplier: 1),
+            titleLabel.topAnchor.constraint(greaterThanOrEqualToSystemSpacingBelow: imageViewContainer.bottomAnchor, multiplier: 1),
+            
+            titleLabel.leadingAnchor.constraint(equalTo: imageViewContainer.leadingAnchor, constant: Self.TextHorizontalMargin),
+            titleLabel.trailingAnchor.constraint(equalTo: imageViewContainer.trailingAnchor, constant: -Self.TextHorizontalMargin),
+            
+            summaryLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: Self.SummaryTopMargin),
+            summaryLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor, constant: Self.TextHorizontalMargin),
+            summaryLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor, constant: -Self.TextHorizontalMargin),
+            
+            findOutMoreButton.topAnchor.constraint(equalTo: summaryLabel.bottomAnchor, constant: Self.FindOutMoreButtonVerticalMargin),
+            findOutMoreButton.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+            findOutMoreButton.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor),
+            findOutMoreButton.heightAnchor.constraint(equalToConstant: Self.FindOutMoreButtonHeight),
+            
+            findOutMoreButton.bottomAnchor.constraint(equalTo: container.bottomAnchor, constant: -Self.FindOutMoreButtonVerticalMargin)
+        ]
+        
+        NSLayoutConstraint.activate(allConstriants)
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        setNeedsUpdateConstraints()
+    }
+
+    func configure(with petition: PresentedPetition) {
+
+        bannerImageView.image = petition.image
+        verifiedLabel.isHidden = !petition.isVerified
+        titleLabel.text = petition.title
+        summaryLabel.text = petition.summary
+        
+        setNeedsUpdateConstraints()
+        setNeedsLayout()
+    }
+
+    @objc
+    private func findOutMoreButtonTapped() {
+        findOutMoreAction()
+    }
+}
+
+// MARK: - Constants
+extension PetitionTableViewCell {
```

Comment:
May consider marking this `private` too

Reply:
Make the Constants private?  In my last PR, people were asking that I put all constants in a Theme.swift file.  I even added a card on Trello (Add a Theme.swift file to collect all UI-related constants). 🙃 For now I'd like to leave it, but I understand why you're saying this. I just don't want to go around and around until these kind of policy things are worked out.

Thanks